lib/app_profiler/sampler/vernier_config.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module AppProfiler
+  module Sampler
+    class VernierConfig
+      attr_reader :modes_probability
+
+      WALL_INTERVAL = 5000
+      RETAINED_INTERVAL = 5000
+
+      WALL_MODE_PROBABILITY = 0.8
+      RETAINED_MODE_PROBABILITY = 0.2
```

Comment:
this is probably too high, the impact of retained profiling is orders of magnitude more than stackprof object profiling. I think the default for this should be 0, and we can assume unless people asked for retained profiling the want wall mode.

In the future we can also consider a third "mode", which is "allocation_sample_rate" done in wall mode, seems to be some kind of a hybrid of wall and stackprof object profile?